### ISSUE TITLE
TCP6ServerEndpoint.__init__'s docstring has a small error

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8209]
|<img alt="Alex's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| Alex reported|
|-|-|
|Trac ID|trac#8209|
|Type|defect|
|Created|2016-02-15 18:45:20Z|

https://github.com/twisted/twisted/blob/trunk/twisted/internet/endpoints.py#L499-L509

The docstring for `__init__` claims that the default value of `interface` is `''`, however the actual default value is `'::'`

<details><summary>Searchable metadata</summary>

```
trac-id__8209 8209
type__defect defect
reporter__Alex Alex
priority__normal normal
milestone__None None
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__easy easy
time__1455561920594541 1455561920594541
changetime__1474198049187875 1474198049187875
version__None None
owner__GitHub__noreply_____ GitHub <noreply@...>

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)|@glyph set owner to @glyph|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="acabhishek942's avatar" src="https://avatars.githubusercontent.com/u/7778186?s=50" width="50" height="50">](https://github.com/acabhishek942)<a name="note_3"></a>|@acabhishek942 removed owner|
|-|-|

Link to PR - https://github.com/twisted/twisted/pull/208

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_4"></a>|GitHub set owner to GitHub<br>GitHub set status to `closed`|
|-|-|

In changeset 5c447e3663c5aa405bddff7da5577ca1fd81794d
```
#!CommitTicketReference repository="" revision="5c447e3663c5aa405bddff7da5577ca1fd81794d"
Merge pull request #295 from twisted/8209-acabhishek942-default

Author: acabhishek942
Reviewer: rodrigc
Fixes: #8209

Fix docstring to reflect correct default interface value
```

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_5"></a>|GitHub commented|
|-|-|

In changeset 24779746cb1aa37f612a96f712a53d92f6a65fd7
```
#!CommitTicketReference repository="" revision="24779746cb1aa37f612a96f712a53d92f6a65fd7"
Merge pull request #296 from twisted/8209-acabhishek942-default-2

Author: rodrigc
Fixes: #8209

Rename 8209.doc to 8209.misc
```

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_6"></a>|@hawkowl commented|
|-|-|

[mass edit] Removing review from closed tickets.
